external/storm-druid/src/main/java/org/apache/storm/druid/bolt/DruidBeamBolt.java

```diff
@@ -76,30 +77,33 @@ public void prepare(Map stormConf, TopologyContext context, OutputCollector coll
 
     @Override
     public void execute(final Tuple tuple) {
-      Future future = tranquilizer.send((druidEventMapper.getEvent(tuple)));
-        LOG.debug("Sent tuple : [{}]" , tuple);
+        if (TupleUtils.isTick(tuple)) {
```

Comment:
@satishd I agree in principle.  I once experimented with that approach and what I found difficult was declaring the `throws` for each of those methods since bolts can generate all sorts of exceptions.

+1